contents/docs/experiments/exposures.mdx

```diff
@@ -0,0 +1,134 @@
+
+## What is an exposure?
+
+An exposure occurs when a user encounters the part of your product where the experiment is running. This is the moment they become a participant in your experiment and start contributing to your metrics.
+
+**For most users**: If you're using PostHog feature flags with our SDKs, exposures are tracked automatically. When you call `getFeatureFlag()`, PostHog sends a `$feature_flag_called` event with all the necessary properties. You don't need to do anything extra.
+
+**Technical details**: PostHog considers a user exposed when it receives a `$feature_flag_called` event containing:
+- The property `$feature_flag` matching your experiment's flag key
+- The property `$feature_flag_response` with a variant value (e.g., "control" or "test")
+
+Only exposed users are included in your experiment analysis. Any events that occur before exposure are ignored, ensuring clean and accurate results.
+
+<CalloutBox icon="IconInfo" title="Important: Exposure comes first" type="info">
+
+Metric events are **only** counted if they occur **after** a user's first exposure. This ensures that:
+- You're measuring the actual impact of your experiment changes
+- Pre-exposure behavior doesn't contaminate your results
+- Each user's journey is measured from the same starting point
+
+For example, if a user makes a purchase before being exposed to your pricing experiment, that purchase won't count toward the experiment metrics.
+
+</CalloutBox>
+
+## Exposure visualization
+
+The experiment view displays real-time exposure data to help you monitor participation:
+
+<ProductScreenshot
+  imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/exposure_2_light_354254b3e7.png"
+  imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/exposure_2_dark_7c1c480e54.png"
+  classes="rounded"
+  alt="Screenshot of experiment exposures"
+/>
+
+The exposures chart shows:
+- **Daily cumulative count**: Unique users exposed to each variant over time
+- **Total exposures**: The absolute number of users in each variant
+- **Distribution percentage**: How users are split across variants
+- **Exposure criteria**: The event being used to determine exposure
+
+## Custom exposure events
+
+While the default `$feature_flag_called` event works for most experiments, you might need a custom exposure event when:
+
+- You are using a different feature flag SDK that doesn't send `$feature_flag_called` events
+- You want more precise control over when users are considered exposed
+- You're running server-side experiments with custom instrumentation
+
+### Setting up custom exposure
+
+To configure a custom exposure event:
+
+1. Click **Edit exposure criteria** in your experiment
+2. Select **Custom** exposure type
+3. Choose your exposure event (e.g., "viewed_checkout_page", "api_endpoint_called")
+4. Add any property filters to refine the exposure criteria
+
+<ProductScreenshot
```

Comment:
Same node screenshot comment as above. It'll even help you keep a consistent border around the element you're screenshotting :)